mlibrary/pride
# Restoring `Pride.requestRecord` as error was thrown from Prejudice.

# Overview
`Pride.requestRecord` was deleted without taking a look at Prejudice, which ended up throwing an error as the method is used in:
* [`./src/Prejudice/Driver/Record/UidHash.js`](https://github.com/mlibrary/prejudice/blob/master/src/Prejudice/Driver/Record/UidHash.js#L16)
* [`./src/Prejudice/Driver/Record/ReactHash.js`](https://github.com/mlibrary/prejudice/blob/master/src/Prejudice/Driver/Record/ReactHash.js#L16)

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#restoring-pride-requestrecord` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#restoring-pride-requestrecord"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- Make sure the PR is consistent in these browsers:
